README_AUTOMATED_RELEASE.md

```diff
@@ -0,0 +1,73 @@
+# Automated release
+
+Make releasing easy!
+
+If you have any question, feel free to ask to me (@wakwa3125).
+
+# ATTENTION
+
+**FIRST YOU MUST UPLOAD YOUR APK FILE WITH NON-AUTOMATED FLOW**  
+**THIS IS PUBLISHING API RESTRICT**
+
+# Setup
+
+## 1. Install supply(fastlane)
+
+```
+gem install fastlane
+```
+
+## 2. Create Service Account at GooglePlayConsole.
```

Comment:
> But there are many noise to read core sentences.

Make sense.
I'm just afraid that it might be hard to follow latest setup info. But it's okay as it is! Thanks!